src/public/modules/core/components/banner.client.component.js

```diff
@@ -26,4 +33,21 @@ function bannerController() {
       vm.bannerHidden = true
     })
   }
+
+  const processBannerMessage = () => {
+    // Retrieve banenr type from message, but it is possible that no types
+    // exist.
+    const vmMessage = vm.message || ''
+    const type = vmMessage.split(':').shift().toLowerCase()
+    const retrievedType = vm.BANNER_TYPES[type]
+
+    vm.bannerType = retrievedType || vm.BANNER_TYPES.info
+
+    // If there is a type retrieved from message, remove the type encoding in
+    // the message.
+    // The + 1 is to also remove the semicolon from the encoding
+    vm.bannerMessage = retrievedType
+      ? vmMessage.substring(retrievedType.length + 1).trim()
```

Comment:
```suggestion
      ? vmMessage.substring(vmMessage.indexOf(':') + 1).trim()
```
Suggest using `indexOf` instead for same reason as above - if there is a space before the semicolon, `retrievedType.length + 1` will not remove the semicolon